docs/reference/commandline/service_create.md

```diff
@@ -761,6 +762,18 @@ $ docker service create --name dns-cache -p 53:53/tcp -p 53:53/udp dns-cache
 $ docker service create --name dns-cache -p 53:53/udp dns-cache
 ```
 
+### Provide credential specs for managed service accounts (Windows only)
+
+This option is only used for services using Windows containers. The
+`--credential-spec` must be in the format `file://spec.txt` or
+`registry://keyname`.
+
+When using the `file://` format, the referenced file must be present on the
+daemon host in the `C:\programfiles\docker\credentialspecs\` directory. When
```

Comment:
It's a relative path from the `C:\programfiles\docker\credentialspecs` folder. Using an absolute path errors correctly:

    time="2017-04-17T08:48:45.126507000-07:00" level=error msg="fatal task error" error="starting container failed: invalid credential spec - file:// path cannot be absolute" module="node/agent/taskmanager" node.id=y6t3hh97zaqh3wkxnrnkg6plw service.id=l5bz9jvh9qxccxdladbg7uec2 task.id=0s3vbr0luxotquovs2ul3fosr

Subdirs can be used, eg. I just tested `C:\ProgramData\docker\credentialspecs\subdir` and it works great.

@aluzzardi let me know if you want to work on the wording or whether you want to do it.